src/main/java/com/lothrazar/storagenetwork/util/RequestBatch.java

```diff
@@ -23,4 +24,40 @@ public void extractStacks(IConnectableLink providerStorage, Integer slot, Item i
         }
         put(item, remainingRequests);
     }
+
+    public void sort() {
+        Collection<List<Request>> requests = this.values();
+        for(List<Request> requestList : requests) {
+            quickSort(requestList, 0, requestList.size() - 1);
+        }
+    }
+
+    private void quickSort(List<Request> requestList, int start, int end) {
+        if (start < end) {
+            int partitionInd = partition(requestList, start, end);
```

Comment:
```suggestion
            int partitionInt = partition(requestList, start, end);
```
Should be int right?

Reply:
ind short for index

Reply:
All right. We can leave it like this then.